components/base/fake/fake.go

```diff
@@ -21,20 +21,14 @@ func init() {
 	resource.RegisterComponent(
 		base.API,
 		resource.DefaultModelFamily.WithModel("fake"),
-		resource.Registration[base.Base, resource.NoNativeConfig]{
-			Constructor: func(
-				ctx context.Context,
-				deps resource.Dependencies,
-				conf resource.Config,
-				logger golog.Logger,
-			) (base.Base, error) {
-				return NewBase(ctx, conf)
-			},
-		},
+		resource.Registration[base.Base, resource.NoNativeConfig]{Constructor: NewBase},
 	)
 }
 
-const defaultWidth = 600
+const (
+	defaultWidthMm               = 600
+	defaultMinimumTurningRadiusM = 0.8
```

Comment:
Maybe I'm being too nitpicky, but why is one of these millimeters and the other meters? Would it make sense to use the same units on both? Maybe not; maybe leave as-is.

Reply:
Nah, let's keep it as-is for the current PR. You had intentionality behind this, and if we want to clean up the messiness, that can be its own PR later.